src/Helper/Scripts.php

```diff
@@ -116,36 +111,39 @@ public function addCondInternal($cond, $script, $pos = 100)
     /**
      * Adds internal script
      *
-     * @param int $pos The script position in the stack.
+     * @param int   $pos  The script position in the stack.
+     * @param array $attr The additional attributes
      *
-     * @return Scripts
```

Comment:
I think this is correct for a method that returns nothing, right?
https://phpdoc.org/docs/latest/references/phpdoc/tags/return.html